drf_dynamic_fields/__init__.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, *args, **kwargs):
             warnings.warn('Context does not have access to request')
             return
 
+        # If the request object does not have query_params attribute, return
+        if not hasattr(self.context['request'], 'query_params'):
+            return
+
```

Comment:
@dbrgn perhaps the better change would be to get fields from either the `query_params` or the `GET` attribute.